src/main/java/com/entropyteam/entropay/employees/models/Employee.java

```diff
@@ -16,18 +17,24 @@ public class Employee extends BaseEntity {
     private String internalId;
     private String firstName;
     private String lastName;
-    private String email;
+    private String personalEmail;
     private String phone;
     private String address;
     private String city;
     private String state;
     private String zip;
     private String country;
     private LocalDate birthDate;
+    private String personalNumber;
+    private String taxId;
+    private String emergencyContactFullName;
+    private String emergencyContactPhone;
 
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "company_id")
-    private Company company;
+    @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
Please take a look at: https://vladmihalcea.com/the-best-way-to-use-the-manytomany-annotation-with-jpa-and-hibernate/

Review why you are fetching it EAGER, as a rule of thumb, never use EAGER
Review the CascadeType

Reply:
Good catch @reconeazaretzky , i recommended use of EAGER, because there was an error related to Role entities, they were not present when we pull the employees. any thoughts?